config/install/rdf_entity.mapping.rdf_entity.spdx_licence.yml

```diff
@@ -0,0 +1,66 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - rdf_entity.graph.default
+    - rdf_entity.rdfentity.spdx_licence
+id: rdf_entity.spdx_licence
+entity_type_id: rdf_entity
+bundle: spdx_licence
+rdf_type: 'http://spdx.org/rdf/terms#License'
+graph:
+  default: 'http://joinup.eu/spdx_licence/published'
+base_fields_mapping:
+  rid:
+    target_id:
+      predicate: 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type'
+      format: resource
+  uid:
```

Comment:
I wonder why we're adding the non-mappings. They're only cluttering the file, making it less readable.